bluepyopt/deapext/algorithms.py

```diff
@@ -90,6 +98,10 @@ def eaAlphaMuPlusLambdaCheckpoint(
         cxpb(float): Crossover probability
         mutpb(float): Mutation probability
         ngen(int): Total number of generation to run
+        stagnation(int): number of evals after which to stop if the fitness
```

Comment:
Thinking about this. Wouldn't it be better to implement a 'stop_criterion' class or something. Then that could have different types, like stagnation. And all the parameters of the stop criterion would be hidden inside the class.

Reply:
Yes I thought about it. Especially since CMA heavily relies on termination criteria. I will try to come up with something.

On of the issue is that the termination criteria for CMA ask for very different information and thus it will be difficult to write general class.